contracts/eosio.system/src/voting.cpp

```diff
@@ -110,8 +110,8 @@ namespace eosiosystem {
       auto constexpr one_percent_weight = total_weight * 0.01;
       auto constexpr num_standbys = 36;
 
-      // multiplied by 21 because we are effectively making 21 separate random selection to insert a standby in this round
-      auto constexpr standby_weight = 21 * one_percent_weight / num_standbys; 
+      // multiplied by 10 (120/12, 12 blocks in 120 seconds)
```

Comment:
this mioght be more accurate: `multiplied by 10 (120/12, 10 rounds of 12 blocks in 120 seconds)`